cyhy/core/geoloc.py

```diff
@@ -39,3 +40,13 @@ def lookup(self, ip):
             return (response.location.longitude, response.location.latitude)
         except AddressNotFoundError:
             return (None, None)
+
+    def check_restricted_ip(self, ip):
```

Comment:
This makes the code more Pythonic, at (IMHO) the expense of clarity.  I'm fine either way.

Reply:
It would simplify things having just one return instead of two. I didn't realize the empty string/dict would come up as false, so I think we can change it that way.